fine-tuning/models.tsp

```diff
@@ -193,20 +244,106 @@ model FineTuningJob {
    * API](/docs/api-reference/files/retrieve-contents).
    */
   validation_file: string | null;
+
+  /** A list of integrations to enable for this fine-tuning job. */
+  // !no_nullable! @maxItems(5)
+  integrations?: FineTuningIntegration[] | null;
+
+  /** The seed used for the fine-tuning job. */
+  seed: int32;
```

Comment:
```suggestion
  seed: int32;
  
  /**
   * The Unix timestamp (in seconds) for when the fine-tuning job is estimated to finish.
   * The value will be null if the fine-tuning job is not running. 
   */
  @encode("unixTimestamp", int32)
  estimated_finish: utcDateTime;
```

missing property `estimated_finish`

Reply:
Ah, that must be a new one! I don''t see it in [the snap from the readme](https://[raw.githubusercontent.com/openai/openai-openapi/2954f15496192a46425313c77713680f89ef85f8/openapi.yaml](https://raw.githubusercontent.com/openai/openai-openapi/2954f15496192a46425313c77713680f89ef85f8/openapi.yaml)), and that wasn't all that long ago...

Reply:
New snap caught it, looks like.